cloud/linode/client/client.go

```diff
@@ -1,6 +1,6 @@
-package linode
+package client
```

Comment:
Just a heads up: moving this file into a new package would constitute a major version bump of this module. However, this module's last-tagged version is still < v1, so it should be fine. Just be aware that this may break downstream users of this module.